src/angular-app/languageforge/lexicon/settings/configuration/configuration-advanced-options.component.ts

```diff
@@ -18,6 +24,11 @@ export class AdvancedOptionsConfigurationController implements angular.IControll
     );
   }
 
+  async resetLocalStorage() {
+    await this.editorOfflineCache.deleteAllEntries();
+    await this.commentsOfflineCache.deleteAllComments();
```

Comment:
should we reload the page after the reset? That'll safely clear any memory that's still holding on to data that was just cleared and it'll start loading the dictionary again.

Reply:
It seemed OK to me. When you navigate back to the editor it downloads the dictionary.
I couldn't break it by going to other configuration tabs either.

So, I think the implemented feature is working well 👍 🚀.

However, there's (aaaalmost) zero feedback when you click the button.
The simplest way to get feedback would be (1) an alert or (2) just reload the page as Kevin suggested, as a result the user will see the dialog indicating that the dictionary is being downloaded, which is probably more meaningful and satisfying than an alert. So, I think reloading the page is a good idea.

I'm sure you could start the download without reloading the page, but if it takes more than 2 minutes to implement it's probably not worth it 😆.

Can we also put a pointer cursor on that button? Pretty please 😁

Reply:
Done. The reset now navigates back to the project root then does a browser reload, so that the redownload is automatically triggered.